pkg/skaffold/test/custom/custom.go

```diff
@@ -83,6 +92,7 @@ func (ct *Runner) Test(ctx context.Context, out io.Writer, artifacts []build.Art
 		return cmdRunRetrieveErr(command, ct.imageName, err)
 	}
 
+	event.TestInProgress()
```

Comment:
shouldn't `TestInProgress` be raised at the start of the method? Otherwise consider it returns on a previous line error. Then you'd have `TestFailed` event without ever having the `TestInProgress` event.

Reply:
Moved it to start of the method.